pages/api/bookings/[screeningid].js

```diff
@@ -3,7 +3,11 @@ import dbConnect from "../../../lib/dbConnect";
 import Screening from "/models/Booking.js";
 
 export default async function handler(req, res) {
-  // await dbConnect();
+  // mongoose.connect(
```

Comment:
If dbConnect() works to connect to Christer kinoDB then there should be no problem.
If not, comment out <await dbConnect()> and use mongoose.connect() to christers kinoDB, link in env.local. or on Discord

Reply:
Works fine with dbconnect.